.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    semi: true,
+    singleQuote: true,
+    tabWidth: 4,
```

Comment:
What is the reason to change from 2 spaces to 4?
I vote for keeping it as is ie. 2 spaces

Reply:
Guess it is a matter of preference. I think code is way more readable that way. Have reverted it to 2 though.